src/js/wizards/Sell/4_Contact/index.jsx

```diff
@@ -33,8 +33,10 @@ class Contact extends Component {
     if (!this.props.seller.location) {
       return this.props.wizard.previous();
     }
+
     if (this.props.profile && this.props.profile.username) {
       this.props.setContactInfo({username: DOMPurify.sanitize(this.props.profile.username), statusContactCode: DOMPurify.sanitize(this.props.profile.statusContactCode)});
+      return this.props.wizard.next();
```

Comment:
Hopefully this doesn't bring back the infinite loop... that was the thing that created it IIRC

Reply:
oh no! :O
yeah, this needs to be tested on a mobile device.